obspy/imaging/waveform.py

```diff
@@ -99,7 +99,7 @@ def __init__(self, **kwargs):
         self.sect_offset_min = kwargs.get('offset_min', None)
         self.sect_offset_max = kwargs.get('offset_max', None)
         self.sect_dist_degree = kwargs.get('dist_degree', False)
-        self.return_ax = kwargs.get('return_ax', False)
+        self.sect_ax = kwargs.get('sect_ax', None)
```

Comment:
Waveform Plotting object is a monster from the dark, early days of ObsPy, so I guess one more option to the gazillion in there makes no difference. The new option should be documented though in `Stream.plot()` docstring. Also `sect_ax` will be a single `Axis` object looks like? In that case I would also do this maybe:

```python
if self.sect_ax is not None:
    self.fig_obj = self.sect_ax.figure
    msg = "Option 'sect_ax' overrides option 'fig'."
    warnings.warn(msg)
```